policy/lib/tekton/task.rego

```diff
@@ -126,14 +126,32 @@ _task_params(task) := params if {
 # task_param returns the value of the given parameter in the task.
 task_param(task, name) := _task_params(task)[name]
 
+# slsa v0.2 results
+_task_results(task) := task.results
+
+# slsa v1.0 results
+_task_results(task) := task.status.taskResults
+
 # task_result returns the value of the given result in the task.
 task_result(task, name) := value if {
-	some result in task.results
+	some result in _task_results(task)
 	result_name := _key_value(result, "name")
 	result_name == name
 	value := _key_value(result, "value")
 }
 
+# slsa v0.2 task steps
+task_steps(task) := task.steps
+
+# slsa v1.0 task steps
+task_steps(task) := task.status.taskSpec.steps
```

Comment:
I must be missing something. I don't see `status` for a task in v1.

Reply:
In the new buildType, the taskRun is stored under resolvedDependencies in the `content` field. This information is taken from there. Here is an example. https://gist.github.com/joejstuart/dce324211434b19990b50e2efe83f548#file-gistfile1-txt-L198